qgroundcontrol.pro

```diff
@@ -1403,3 +1403,39 @@ LinuxBuild {
 
     INSTALLS += target share_qgroundcontrol share_icons share_metainfo share_applications
 }
+
+
+#-------------------------------------------------------------------------------------
+#For 3D Viewer
+
+QT += \
+    quick3d \
+    xml
+
+CONFIG += qmltypes
```

Comment:
I'm not quite sure what this is doing fully. I think it is adding a new qml module. If so, it's not compiling on my Mac. Can you change this to setup up the module like the other QGC qml modules.

Reply:
If you mean "CONFIG += qmltypes", yes, indeed, it is the new way (as opposed to qmlRegisterType) of defining and registering QML Types from C++ (based on this reference: https://doc.qt.io/qt-6/qtqml-cppintegration-definetypes.html).
I can change them to the conventional way by using "qmlRegisterType".

But please also note that to be able to compile the code, **"Qt Quick 3D"** needs to be installed using the Qt Maintenance Tool as I highlighted in the figure below:

![image](https://github.com/mavlink/qgroundcontrol/assets/115142498/c58f5763-021e-45f5-8c47-36db2b8c5321)